sqlx-core/src/sqlite/connection/mod.rs

```diff
@@ -40,6 +43,15 @@ impl SqliteConnection {
     pub fn as_raw_handle(&mut self) -> *mut sqlite3 {
         self.handle.as_ptr()
     }
+
+    pub fn create_collation(
+        &mut self,
+        name: &str,
+        collation: impl Fn(&str, &str) -> Ordering + Send + Sync + UnwindSafe + 'static,
+    ) -> &mut SqliteConnection {
+        collation::create_collation(&self.handle, name, collation);
```

Comment:
Please bubble the error here and return `Result<(), Error>`. We shouldn't need this to enable chaining.